microsoft/PowerToys
# [PTRun] add support for UUIDv7 generation

## Summary of the Pull Request


## PR Checklist

- [x] **Closes:** #33266
- [x] **Communication:** ~I've discussed this with core contributors already. If work hasn't been agreed, this work might be rejected~ -> was marked as [Help Wanted](https://github.com/microsoft/PowerToys/labels/Help%20Wanted)
- [x] **New binaries:** ~Added on the required places~ -> nothing to add
- [x] **Documentation updated:** If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/windows-uwp/tree/docs/hub/powertoys) and link it here: https://github.com/MicrosoftDocs/windows-dev-docs/pull/5206


## Detailed Description of the Pull Request / Additional comments

This PR implements UUIDv7 generation for PowerToys Run as defined in [RFC 9562](https://www.rfc-editor.org/rfc/rfc9562#name-uuid-version-7).


## Validation Steps Performed

Added unit tests to ensure the version field is correctly set and to assert subsequently generated UUIDs are time-ordered. Performed manual testing using locally built x64 version of PT Run to ensure correct behavior.
